easybuild/easyconfigs/i/imkl/imkl-2018.3.222-gompi-2018b.eb

```diff
@@ -0,0 +1,36 @@
+# This is an easyconfig file for EasyBuild, see https://easybuilders.github.io/easybuild/
+
+name = 'imkl'
+version = '2018.3.222'
+
+homepage = 'https://software.intel.com/en-us/intel-mkl/'
+description = """Intel Math Kernel Library is a library of highly optimized,
+ extensively threaded math routines for science, engineering, and financial
+ applications that require maximum performance. Core math functions include
+ BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more."""
+
+toolchain = {'name': 'gompi', 'version': '2018b'}
+
+sources = ['l_mkl_%(version)s.tgz']
```

Comment:
@akesandgren `source_urls` is also missing, which we've started including in `imkl` easyconfigs recently...

```python
source_urls = ['http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/13005/']